scripts/zones/Beadeaux_[S]/MobIDs.lua

```diff
@@ -0,0 +1,9 @@
+-----------------------------------
+-- Area: Beadeaux [S] (92)
+-- Comments: -- posX, posY, posZ
+-- (Taken from 'mob_spawn_points' table)
+-----------------------------------
+
+Da_Dha_Hundredmask = 17154195; -- -89.901, .225, -159.964, 92
+Da_Dha_Hundredmask_PH = 
+{[17154095] = '1',};  -- Gold Quadav -37, 1, -128, 92
```

Comment:
Is that the only Gold Quadav in the area? 

I realize retail only uses one PH but because we lack multipel spawn points that multiple spawn ID's can pop into, we've bene using multipel ID's for teh PH to compensate since forever. Like take leaping lizzy, on retail it will always be the same mob on widescan, but in DSP we use:

```lua
-- Leaping Lizzy
Leaping_Lizzy=17215888;
Leaping_Lizzy_PH =
{
  [17215864] = '1', -- -228.159, 18.045, -411.726
  [17215863] = '1', -- -260.662, 22.417, -357.882
  [17215865] = '1', -- -270.019, 20.612, -389.970
  [17215867] = '1', -- -275.441, 20.451, -347.294
  [17215866] = '1', -- -276.306, 20.355, -345.704
  [17215887] = '1', -- -322.871, 30.052, -401.184
  [17215885] = '1', -- -327.621, 31.526, -410.456
  [17215886] = '1', -- -376.273, 32.776, -441.078
  [17215884] = '1', -- -362.393, 29.985, -393.530
  [17215883] = '1'  -- -341.113, 32.862, -333.188
};
```
Which is every lizard in lizzies 2 spawn groups.

If that is the only Gold Quadav nearby then its fine as is, though the inline bracing and the trailing comma when no element follows threw me for a second.